client/src/app/filter-and-sort/filter-and-sort.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FilterSortOptions } from '@services';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'app-filter-and-sort',
+  templateUrl: './filter-and-sort.component.html',
+  styleUrl: './filter-and-sort.component.scss',
+})
+export class FilterAndSortComponent {
```

Comment:
Make this a sub-component of `recipe-search`, move the component dir under `recipe-search`.